### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Peter1907**
Hello, dear code reviewer!

### In this branch, I've implemented the following:

- Updated the actions inside the user & post controller to include the instance variables needed
- Updated the index & show views to include the required info for both the user & post controllers.

Let me know if there's anything to be modified or improved.
Thank you so much for your time & effort.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,10 @@
...
\ No newline at end of file
+<h1>List of all users</h1>
+<% @users.each do |user| %>
+  <div class="user">
+    <img src=<%= user.photo %> alt="user pic" />
+    <div class="title">
+      <h3><%= link_to user.name, user_path(user) %></h3>
+      <p>Number of posts: <%= user.posts_counter %></p>
+    </div>
+  </div>
```
Comment (Munsa1) [inline app/views/users/index.html.erb]: - [ ] Kindly follow the given design for the `users` page as this is a requirement. This means following the layout of components as is in the given wireframes

Given design
image

Your design
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,24 @@
...
+  </div>
+  <div class="bio">
+    <h4>Bio:</h4>
+    <p><%= @user.bio %></p>
+  </div>
+  <div class="posts">
+    <% @user.last_3_posts.each_with_index do |post, i| %>
+      <div class="post">
+        <h4><%= link_to "Post #" + (i + 1).to_s, user_post_path(@user, post) %></h4>
+        <p><%= post.text %></p>
```
Comment (Munsa1) [inline app/views/users/show.html.erb]: - [ ] Kindly follow the given wireframe template for each user, this includes layout of components on the page as this is a requirement.

Given design
image

Your design
image

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -1 +1,12 @@
...
+    <h3>Post by <%= @post.author.name %></h3>
+    <p>Commets: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+  </div>
+  <p class="post-body"><%= @post.text %></p>
+  <div class="comments">
+    <% @post.comments.each do |comment| %>
+      <p><%= comment.author.name %>: <%= comment.text %></p>
+    <% end %>
+  </div>
+</div>
```
Comment (Munsa1) [inline app/views/posts/show.html.erb]: - [ ] Again here, kindly follow the given design, this includes the border, positioning and layout of components as this is a requirement

Given wireframe
image

Your design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Peter1907, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @Peter1907       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: aa
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R








 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**